Jenkinsfile

```diff
@@ -57,7 +59,7 @@ pipeline {
         pipelineManager([ cancelPreviousRunningBuilds: [ when: 'PR' ] ])
         deleteDir()
         gitCheckout(basedir: "${BASE_DIR}", githubNotifyFirstTimeContributor: true)
-        stash allowEmpty: true, name: 'source', useDefaultExcludes: false
+        stashV2(allowEmpty: true, name: 'source', useDefaultExcludes: false)
```

Comment:
NOTE: if we move forward then the signature for this particular method should be changes, there are certain arguments that are not required at all, such as `useDefaultExcludes`